.github/workflows/publish-charm.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
           built-charm-path: ${{ inputs.charm-file-name }}
           credentials: "${{ secrets.CHARMCRAFT_AUTH }}"
           github-token: "${{ secrets.GITHUB_TOKEN }}"
-          channel: latest/edge
+          channel: 1.3/edge
```

Comment:
Those should likely be inputs instead of assuming the channel to be `1.3/edge`

Reply:
This is what I was asking in the daily.  How can the charm being built pass that info to this workflow?

Reply:
Through workflow call inputs, each charm's CI can pass the channel it wants.

Right now this can be an input parameter here and the 1.3/edge be hardcoded in every charm CI workflow but in the future, those individual CI workflows can read from the metadata.yaml's `upstream-source` or from the branch name

Reply:
Set as a variable